<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortPlaylistDialog</class>
 <widget class="QDialog" name="SortPlaylistDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Playlist(s)</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>480</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="playlistList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>781</width>
     <height>221</height>
    </rect>
   </property>
   <property name="selectionRectVisible">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="shuffleCheckBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Shuffle the playlist(s) instead of sorting.</string>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
  </widget>
  <widget class="QListWidget" name="availableAttributesListWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>256</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="selectedAttributesListWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>256</width>
     <height>131</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Rearrange the order of this list to change the sorting priority</string>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::InternalMove</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="reverseCheckbox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Reverse the sort order</string>
   </property>
   <property name="text">
    <string>Reverse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>751</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select one or more playlists to sort:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>275</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sortable attributes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>275</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected attributes (drag to rearrange)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SortPlaylistDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SortPlaylistDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
